docs/running-a-node/ledger-management.md

```diff
@@ -187,39 +142,35 @@ This can be enabled by adding the following to the `config-node.toml` file:
 enable = true
 ```
 
-There are many other options which can be set. Due to RocksDB generally using more memory the defaults have been made pessimistic in order to run on a wider range of lower end devices. Recommended settings if on a system with 8GB or more RAM (see TOML comments in the generated file for more information on what these do):
-
+The other options are:
 ```
-[node.rocksdb]
-bloom_filter_bits = 10
-block_cache = 1024
-enable_pipelined_write=true
-cache_index_and_filter_blocks=true
-block_size=64
-memtable_size=128
-num_memtables=3
-total_memtable_size=0
+io_threads = 4
+memory_multiplier = 2
 ```
-Comparision:
+It shouldn't be necessary to update these variables manually. See TOML comments in the generated file for more information on what these do.
+
+### Migrating existing ledger from LMDB to RocksDB
+An existing LMDB ledger can be upgraded by running the [--migrate_database_lmdb_to_rocksdb](/commands/command-line-interface/#-migrate_database_lmdb_to_rocksdb) CLI command. This process can take some time, estimates range from 20 minutes to 1 hour depending on node hardware specs. There are some internal checks which are made to determine if the migration was successful, however it is recommended to run the node first (after enabling RocksDB) for a period of time to make sure things are working as expected. After which the `data.ldb` file can be deleted if no longer required to save on disk space. Please also note the [limitations](#rocksdb-limitations) most notably is that the `unchecked_count` from the `block_count` RPC will only be an estimate.
```

Comment:
- [ ] Add more emphasis on the requirement to manually enable RocksDB after migration - might not be obvious that extra step is needed